apps/nextjs-website/package.json

```
@@ -16,7 +16,7 @@
     "@mui/icons-material": "^5.14.14",
     "@mui/material": "^5.14.14",
     "@pagopa/mui-italia": "^1.0.1",
-    "@pagopa/pagopa-editorial-components": "^2.3.1",
+    "@pagopa/pagopa-editorial-components": "^2.3.4",
```

```suggestion
    "@pagopa/pagopa-editorial-components": "^2.3.2",
```
I think this is the version you need